test/system/120-load.bats

```diff
@@ -147,4 +147,43 @@ verify_iid_and_name() {
        "Diagnostic from 'podman load' without redirection or -i"
 }
 
+@test "podman load - multi-image archive" {
+    alpine="docker.io/library/alpine:latest"
+    busybox="docker.io/library/busybox:latest"
```

Comment:
We no longer use docker.io - we can't. We use `quay.io/libpod/testimage:<some tag>`. I believe you could accomplish this with
```bash
    img1="quay.io/libpod/testimage:00000000"
    img2="quay.io/libpod/testimage:20200902"
```
Both have been used by prior system tests, hence need to exist forever. If you actually need images that differ in name as well as tag, you could use `testdigest_v2s2:20200210` or `:latest`. That is a tiny image that should be quick and safe to pull.

Or, since one of my top goals in system tests is to minimize pulls (because they're a cause of flakes), you could instead `podman build` with custom tags.

Reply:
Thanks, Ed! I went with pulling.